### PR TITLE
#165279481 Update and delete an article

### PR DESCRIPTION
**Pull request by shaolinmkz**
#### What does this PR do?
Have the edit and delete article feature implemented.

#### Description of Task to be completed?
1. Consume the edit article endpoint
2. consume the delete article endpoint

#### How should this be manually tested?
1. Clone this branch.
2. Run `cd artemis-ah-frontend`
3. Run `npm install` to install the dependencies.
4. Run `npm test` to run the tests.

#### What are the relevant pivotal tracker stories?
[#165279481](https://pivotaltracker.com/story/show/165279481)

#### Screenshots (if appropriate)
Screenshot 2019-04-23 at 12 03 01 PM
Screenshot 2019-04-23 at 12 03 15 PM

### CODE REVIEW

**Review by uniqueayo1988 — COMMENTED** [review]
Reviewed.
LGTM

**Review by Chrismarcel — COMMENTED** [review]
LGTM

**Review by Dsalz — COMMENTED** [review]
Reviewed

File: src/views/ProfilePage.jsx [hunk 1]
```diff
@@ -146,27 +152,13 @@ class ProfilePage extends Component {
...
-
-            const text = event.clipboardData.getData('text/plain').substr(0, 190);
-            document.execCommand('insertHTML', false, text);
-          };
-
-          aboutProps.onKeyDown = (event) => {
-            if (event.target.innerText.length > 190 && event.keyCode !== 8) {
-              event.preventDefault();
-            }
-          };
```
Comment (danprocoder) [inline src/views/ProfilePage.jsx]: Why did you remove this block of code?

File: src/views/ProfilePage.jsx [hunk 2]
```diff
@@ -339,9 +352,11 @@ class ProfilePage extends Component {
     if (user.isLoggedIn && user.username === profile.user.username) {
       return (
         <div className="no-result">
-          <div className="no-result__msg">You don&apos;t have any article.</div>
```
Comment (danprocoder) [inline src/views/ProfilePage.jsx]: Why did you remove the className here?

**Review by andela-gike — DISMISSED** [review]

File: src/views/ProfilePage.jsx (hunk 2 above)
Comment (shaolinmkz) [reply src/views/ProfilePage.jsx]: Resolved

File: src/views/ProfilePage.jsx (hunk 1 above)
Comment (shaolinmkz) [reply src/views/ProfilePage.jsx]: Resolved

**Review by andela-gike — APPROVED** [review]
